### PR TITLE
libshumate 1.0.5 (new formula)

### PR DESCRIPTION
**Pull request by taozuhong**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/libshumate.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => [:build, :test]
+  depends_on "vala" => :build
+  depends_on "gtk4"
+  depends_on "libsoup"
+  depends_on "sqlite"
+
+  def install
+    system "meson", "setup", "build", "-Dlibsoup3=true" *std_meson_args
```
Comment (alebcay) [inline Formula/libshumate.rb:29]:
```suggestion
    system "meson", "setup", "build", "-Dlibsoup3=true", *std_meson_args
```

Hello 👋 thanks for opening a pull request! Missing a comma here, which is causing the `TypeError: no implicit conversion of Array into Integer` error.

File: Formula/libshumate.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  bottle do
+    sha256 arm64_ventura:  "46cd0d4f81a5c2d5efc63b1755a7a002ff95fa8f42007668a337327dd9970607"
+    sha256 arm64_monterey: "9550badfd2988da515c5ff5557014fa3d51a99e4880dd1d20412dc913e57aa2b"
+    sha256 arm64_big_sur:  "0bcf580842dc04a29575b36747ff63ec4456e93dca113be677b2efc610f0d65d"
+    sha256 ventura:        "d40e0ea4a951ce312ea727f83c1127a0c2790574ddb6e7990d0dd93f5eb6baa1"
+    sha256 monterey:       "acfb1a2fe6e8147e6e2e89fadb5890b34d8e6863d040fa39c068c10eb5d21082"
+    sha256 big_sur:        "72ee89c1212e23161ab35c91faea62ee9ce2fef2e533b2269b63c8ffd0d3a1f1"
+    sha256 x86_64_linux:   "09ffb6416daa2e1367119e976d17d0f7a9c486940eb0b1fe6b42b0b8415fbe0d"
+  end
+
```
Comment (chenrui333) [inline Formula/libshumate.rb:17]:
```suggestion
```

File: Formula/libshumate.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  def install
+    system "meson", "setup", "build", "-Dlibsoup3=true" *std_meson_args
+    system "meson", "compile", "-C", "build", "--verbose"
+    system "meson", "install", "-C", "build"
+  end
+
+  test do
+    # Remove when `jpeg-turbo` is no longer keg-only.
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["jpeg-turbo"].opt_lib/"pkgconfig"
+
```
Comment (chenrui333) [inline Formula/libshumate.rb:37]:
```suggestion
```

**Comment by taozuhong** [comment]
@chenrui333 Thank awfully.

File: Formula/libshumate.rb [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      int main(int argc, char *argv[]) {
+        const char* agent = shumate_get_user_agent();
+        return 1;
+      }
+    EOS
+    ENV.prepend_path "PKG_CONFIG_PATH", Formula["icu4c"].opt_lib/"pkgconfig" if OS.mac?
+    flags = shell_output("#{Formula["pkg-config"].opt_bin}/pkg-config --cflags --libs shumate-1.0").strip.split
+    system ENV.cc, "test.c", "-o", "test", *flags
+    # system "./test", "--help"
```
Comment (SMillerDev) [inline Formula/libshumate.rb:39]:
```suggestion
    system "./test"
```

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
you have to always do the rebase rather than the merge commit, unfortunately

**Comment by chenrui333** [comment]
@taozuhong, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**